manage_breast_screening/notifications/services/nhs_mail.py

```diff
@@ -32,6 +32,16 @@ def send_report_email(
             attachment_data, attachment_filename, report_type
         )
 
+        logger.info(
+            f"Email for report type {report_type} created with attachment {attachment_filename}"
+        )
+
+        if os.getenv("NOTIFICATIONS_SMTP_IS_ENABLED", "False") == "True":
```

Comment:
```suggestion
        if bool(os.getenv("NOTIFICATIONS_SMTP_IS_ENABLED")):
```
is not as explicit but works for `None (False), "" (False), 0 (False), 1 (True), "true" (True), "True" (True)` I'd sleep well either way :laughing:

Reply:
ah thank you - I was sure there was a better way of doing it but my 1 minute of googling didn't return anything :D